### PR TITLE
feat: refresh About page 

### LINKED ISSUE #1063: refresh the About pages
**Please describe the feature**

As a user, I want the About page to be clear and familiar, so that I find what I'm after as easy as possible.

**Acceptance Criteria**

- [x] align the styling between the Documentation and About pages
- [x] condense the sidebar menu for the About pages
- [x] The changes are tested on multiple models and browsers
- [x] The changes do not affect the deployment
- [x] The changes do not reduce the performance

**Additional context**

### PR DESCRIPTION
**Pull request by mihai-sysbio**
**Related issue(s) and PR(s)**  
This PR closes #1063.

**Type of change**  

- [x] New feature (non-breaking change which adds functionality)
 
**List of changes made**  

For the list of changes see the commit messages.

**Testing**  

Navigate to the `About` page and play around.

**Further comments**  
There are other places in the website that link to the `About` pages. Those links should still be functional.

**Definition of Done checklist**  
- [x] My code meets the Acceptance Criteria
- [x] My code follows the [NBIS style guidelines](https://github.com/NBISweden/development-guidelines)
- [x] I have performed a self-review of my own code and commented any hard-to-understand areas
- [x] Tests and lint/format validations are passing
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by MalinAhlberg — APPROVED** [review]
I think this looks very good!

I'm a bit annoyed that if you go to for example `about/platform#advisory-board` and then click `Platform`, it doesn't jump back to the top of the page, but that's a minor issue.

**Review by nanjiangshu — COMMENTED** [review]
The new about page looks great with categories @mihai-sysbio. But I do have a few comments as I marked on the screenshot.
1. The white space between the banner and the title is too large due to the addition of title "About" on the Table of Contents. Moreover, it also creates a constant misalignment of bold titles on the top. I think it might be better to remove the title "About", i.e., looks better in the previous version.
2. I'm not sure if "Terms" is a good word for the category of privacy and licenses. Maybe extend it more, "Terms of use"?
3. The word "Introduction" in the Table of Contents should be "Privacy policy"

File: frontend/src/content/about.js [hunk 1]
```diff
@@ -1,26 +1,28 @@
...
-  { name: 'Citation', icon: 'fa-quote-left', link: '/about/citation', routeName: 'about-citation' },
-  { name: 'Contact us', icon: 'fa-envelope-o', link: '/about/contact', routeName: 'about-contact' },
-  { name: 'Privacy policy', icon: 'fa-shield', link: '/about/privacy', routeName: 'about-privacy' },
+  { name: 'News', icon: 'fa-newspaper-o', link: '/about/news', routeName: 'about-news' },
   {
-    name: 'License and Disclaimer',
-    icon: 'fa-book',
-    link: '/about/license',
-    routeName: 'about-license',
+    name: 'Terms',
```
Comment (mihai-sysbio) [inline frontend/src/content/about.js:18]:
```suggestion
    name: 'Terms of use',
```

File: frontend/src/content/about.js [hunk 2]
```diff
@@ -1,26 +1,28 @@
...
+    icon: 'fa-shield',
+    link: '/about/terms',
+    routeName: 'about-terms',
+    subsections: [
+      { name: 'Introduction', link: '/about/terms#privacy-policy' },
```
Comment (mihai-sysbio) [inline frontend/src/content/about.js:23]:
```suggestion
      { name: 'Privacy policy', link: '/about/terms#privacy-policy' },
```

**Comment by mihai-sysbio** [comment]
Thanks @nanjiangshu  for the review.

> * The white space between the banner and the title is too large due to the addition of title "About" on the Table of Contents. Moreover, it also creates a constant misalignment of bold titles on the top. I think it might be better to remove the title "About", i.e., looks better in the previous version.

This is intentionally copying the layout of the Documentation, so as to avoid titles sitting too close visually to each other and competing for the user's attention. A bit of horizontal hierarchy simplifies things.

> * I'm not sure if "Terms" is a good word for the category of privacy and licenses. Maybe extend it more, "Terms of use"?

Nice, the suggestion is applied.

> * The word "Introduction" in the Table of Contents should be "Privacy policy"

Great job spotting this!

The fixes are in 103988ac69638d5ed50cd8f693c03694cc7a059c.

**Comment by mihai-sysbio** [comment]
> I think this looks very good!
> 
> I'm a bit annoyed that if you go to for example `about/platform#advisory-board` and then click `Platform`, it doesn't jump back to the top of the page, but that's a minor issue.

I see your point @MalinAhlberg. Anchors should then be added at the top of the page.Would you like to have a go at this?

**Review by nanjiangshu — APPROVED** [review]
LGTM!

**Comment by MalinAhlberg** [comment]
> > I'm a bit annoyed that if you go to for example `about/platform#advisory-board` and then click `Platform`, it doesn't jump back to the top of the page, but that's a minor issue.
> 
> I see your point @MalinAhlberg. Anchors should then be added at the top of the page. Would you like to have a go at this?

I did not find any perfect solution for this, but have committed one that is working. Please try it out!

It's not working perfectly on mobile devices, though, but that was the case even before my commits.
To try this, try clicking eg `Contact us` from the Table of Content on a mobile device (scrolls to the `Contact us` section), and compare to clicking `News` (stays on the menu). I propose we create a new issue for this.

**Comment by mihai-sysbio** [comment]
Thanks for fixing this @MalinAhlberg.

It's weird, since there is already an id for `News.vue` that doesn't seem to do the job (after lowercasing it and updating the corresponding link). It might have to do with the content being dynamically generated, so the scrolling happens too soon. 

Since the route names are not used anymore, a cleanup of _about.js_ removing this would make sense to me.

**Comment by MalinAhlberg** [comment]
> Since the route names are not used anymore, a cleanup of _about.js_ removing this would make sense to me.

The route names are still used from the navbar, so simple removing them does not work, unfortunately. But that could be part of the new issue, to look at how to clean things up.

**Pull request merged**